app/renderer/src/assets/locales/en/translation.json

```diff
@@ -282,5 +282,14 @@
   "invalidCapType": "Invalid capability type: {{type}}",
   "whitespaceDetected": "Text Starts and/or Ends With Whitespace",
   "pcloudyCredentialsRequired": "pCloudy username and api key are required!",
-  "duplicateCapabilityNameError": "A capability set with the same name already exists"
+  "duplicateCapabilityNameError": "A capability set with the same name already exists",
+  "errorLoadingGestures": "Error loading gestures",
+  "unableToUploadGestureFiles": "Could not upload gesture from below files:",
+  "gestureEmptyTickError": "Actions should have valid ticks.",
+  "gestureInvalidEventError": "Invalid event type {{invalidEvent}}. Valid values are {{validEvents}}.",
```

Comment:
Lines 289, 290/291, 294 - would suggest placing each parameter in single quotes (e.g. `Invalid event type '{{invalidEvent}}'.` - it would make it easier to understand the sentence.